Jenkinsfile

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env groovy
+library 'status-jenkins-lib@v1.8.8'
```

Comment:
I would recommend using newest tag available in `status-jenkins-lib`, currently `v1.9.4`.